tests/queries/0_stateless/02775_show_columns_mysql_compatibility.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
```

Comment:
Curious why this test can't be a "normal" .sql test?

Reply:
Since we're testing both the clickhouse-client and mysql client simultaneously we need to use a shell script to leverage both tools.